markdown/articles/2021/07/MonkeyWorld.md

```diff
@@ -0,0 +1,193 @@
+---
```

Comment:
Article file path needs day value added I think

Reply:
@angrybacon is this something you can sort during push?

Reply:
Yup, will take care of it